tests/components/line-series-canvas-test.js

```diff
@@ -0,0 +1,51 @@
+import test from 'tape';
+import React from 'react';
+import {mount} from 'enzyme';
+import XYPlot from 'plot/xy-plot';
+import LineSeriesCanvas from 'plot/series/line-series-canvas';
+
+const LINE_PROPS = {
+  className: 'line-chart-example',
```

Comment:
this classname doesnt do anything right? you can remove it

Reply:
i just copy paste props from some different test, will do